File: client/src/autogenerated/a-p-i/activities/models/activity-api-model.ts
import { BaseApiModel } from '../../common/base-api-model';
import { ProfileApiModel } from '../../profiles/models/profile-api-model';

export class ActivityApiModel extends BaseApiModel {
    id?: number;
    title: string;
    date = new Date();
    description: string;
    categoryId: number;
    category: string;
    city: string;
    venue: string;
    host = new ProfileApiModel();
    isCancelled: boolean;
    attendees = new Array<ProfileApiModel>();
}

File: Reactivities.Server/API/Activities/Models/ActivityApiModel.cs
using System;
using System.Collections.Generic;
using API.Common;
using API.Profiles.Models;

namespace API.Activities.Models
{
    public class ActivityApiModel : BaseApiModel
    {
        public int? Id { get; set; }
        
        public string Title { get; set; }
        
        public DateTime Date { get; set; }
        
        public string Description { get; set; }
        
        public int CategoryId { get; set; }

        public string Category { get; set; }

        public string City { get; set; }
        
        public string Venue { get; set; }
        
        public ProfileApiModel Host { get; set; }
        
        public bool IsCancelled { get; set; }

        public ICollection<ProfileApiModel> Attendees { get; set; } = new List<ProfileApiModel>();
    }
}
